pulp_service/pulp_service/app/viewsets.py

```diff
@@ -138,3 +149,42 @@ def get_queryset(self):
     @classmethod
     def view_name(cls):
         return "admintasks"
+
+
+class ContentScan(APIView):
+    authentication_classes = []
+    permission_classes = []
+
+    def get(self, request=None, prn=None):
```

Comment:
From a usability perspective I think the argument here should be `repo_version` instead of `prn` although the value should be in the format `prn`. From an openAPI perspective the way this is currently it's polymorphic in the sense a PRN can refer to lots of different object types in Pulp. To keep things simple I think we should have this refer to a `repo_version` and then this viewset should provide validation that the prn is indeed referring to a repository version.